hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot

```diff
@@ -72,7 +72,7 @@ Test ozone shell
                     Execute             ozone sh bucket clrquota --namespace-quota ${protocol}${server}/${volume}/bb1
     ${result} =     Execute             ozone sh bucket info ${protocol}${server}/${volume}/bb1 | jq -r '. | select(.name=="bb1") | .quotaInNamespace'
                     Should Be Equal     ${result}       -1
-                    Execute             ozone sh bucket delete ${protocol}${server}/${volume}/bb1
+                    Execute             ozone sh bucket delete -r --yes ${protocol}${server}/${volume}/bb1
```

Comment:
Do we need the recursive option added to the bucket delete here? Is there a case where trash is remaining when this keyword is called?

Reply:
Yes because at L62, we are calling `Test key handling`, which deletes the key and key moves inside trash.